assets/css/_footer.scss

```diff
@@ -1,27 +1,61 @@
-.about-footer{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    gap: 1rem;
-    height: 10rem;
-    p{
-        font-family: $lato-family;
-        font-size: 1.4rem;
-    }
+.about-footer {
```

Comment:
- [ ] If we look at the given `footer `design of `about page` in desktop version, we can see it has `a dark background`. But you have it white background. Let's stick to the original design, what do you think?
- Original design 👇 
![-2](https://user-images.githubusercontent.com/67087664/181525291-e24f1b23-01db-4914-9692-fe29d8bce95b.png)
- Your footer design 👇 
![2](https://user-images.githubusercontent.com/67087664/181525385-9fdc1ce6-7b20-4a88-bab0-cb39068f8213.png)